docs/auth/README.md

```diff
@@ -135,19 +137,36 @@ curl -H "Authorization: Bearer $UMS_TOKEN"
 </template>
 </CodeSwitcher>
 
+::: warning
+Keep in mind at this point in time only JS SDK is automatically refresh access token. If you use REST API or Python SDK you need to handle this situation by your own. Token live time 2h 
+:::
+
 ## Policies
-As was mentioned above to authorize access to your data you need to have a **policy** created for your API keys and Project Users. 
+As was mentioned above to authorize access to your data you need to have a **policy** created for your API keys and/or Project Users. 
 Currently, you cannot manipulate a **policy** via the API. All admin actions are only available via Jexia's administration dashboard.
 
 <iframe width="700" height="394" src="https://www.youtube.com/embed/i4dKznoXry0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-There are three main areas within a **Policy**:
-1. **Subject** (who has access) - can be an **API key**, **All Users** (all project users) or a **namespace** (grouped users under the same name).
-2. **Resources** (access to what) - can be any dataset, fileset or channel. Selecting **All Users** means you allow these operations to be performed by all Project Users. 
-3. **Actions** (what CRUD actions can be performed) - here you can specify which actions are allowed to be performed. These can be: Create, Read, Update and Delete. For Channels you have specific actions: Subscribe (read) and Publish (write).
+_* video shows previous version of policy screen_
+
+There are four main areas within a **Policy**:
+1. **Subject** (who has access) - can be an **API key**, **Namespace** (grouped users under the same name), **User account**.
+2. **Resources** (access to what) - can be any dataset, fileset or channel. Selecting **All Users** means you allow  operations for all Project Users. 
+3. **Actions** (what CRUD actions can be performed) - here you can specify which actions are allowed to be performed. These can be: Create, Read, Update and Delete. For Channels you have specific actions: Subscribe (read) and Publish (write). 
+4. **Filters** - for actions you can use additional filtering options to specify access of current namespace, api or user. It is useful when you want to segrigate data access for different instances of your business(offices, e-shops, etc).
+We have predefined filter - Owner which says to the platform to filter out only records where user is an owner. It can combine with additional filters to specify even more. All filters joind with **AND** logical operator. Changes for policy applies immediately.
```

Comment:
Maybe change: "We have predefined filter" to "We have a predefined filter" and: "it can combine with additional filters" to "It can be combined with additional filters".

Reply:
Reviewed by Bailey as well, see the commit history